packages/markdown-docx/src/ToOOXMLVisitor/index.js

```diff
@@ -224,6 +251,14 @@ class ToOOXMLVisitor {
                             this.globalOOXML += ooxml;
                             this.tags = [];
                         } else {
+                            if (this.getClass(subNode) === TRANSFORMED_NODES.link) {
+                                const relationshipTag = `<Relationship Id="rId${
+                                    this.relationships.length + 11
+                                }" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/hyperlink" Target="${
+                                    subNode.destination
+                                }" TargetMode="External"/>`;
+                                this.relationships = [...this.relationships, relationshipTag];
```

Comment:
I think it would be better to just collect `subNode.destination` here. And then you can pass that array to `RELATIONSHIP_SPEC`.

Reply:
@algomaster99 
We would need the `id` also because that defines what connects what.
If we pass it to `RELATIONSHIP_SPEC`, then we would need to construct the tags there eventually. I don't understand the reason for passing it like this.